services/secret-service/stub/src/main/proto/SecretService.proto

```diff
@@ -72,6 +72,34 @@ message S3SecretDeleteRequest {
 }
 
 
+// Box
+
+message BoxSecret {
+    string secretId = 1;
+    string accessToken = 2;
```

Comment:
Does this only has access token based authorization? Is there client id / client secret type authorization? If there is such, I prefer that also be added here

Reply:
I'm working on it.

Reply:
According to their documentation, this is the option available for free. There's an option to generate a JWT token not available for free. How that works is, another managed user (API only user) have to be created who has access to a particular directory and through him only the connection can be established.
https://github.com/box/box-java-sdk/blob/master/doc/authentication.md#developer-token